src/components/DailyRetro/Form/Form.js

```diff
@@ -5,26 +5,9 @@ import SleepSlide from "../CarouselForm/slides/Sleep/SleepSlide.js";
 import BreaksSlide from "../CarouselForm/slides/Breaks/BreaksSlide.js";
 import {
   Carousel,
-  CarouselItem,
-  CarouselControl,
-  CarouselIndicators,
+  CarouselItem
 } from "reactstrap";
 
-//in documentation, items are in an object form? may need to change if we have any trouble
```

Comment:
Love you that reformatted and removed so much code! It really seems to have cleaned up the code base!